CHANGES

```diff
@@ -120,6 +115,123 @@ Tools
   Verbose Compat vernacular, since these warnings can now be silenced or
   turned into errors using "-w".
 
+Bugfixes
+
```

Comment:
A query that can be run without login: https://coq.inria.fr/bugs/buglist.cgi?bug_status=RESOLVED&bug_status=CLOSED&chfield=resolution&chfieldfrom=2016-01-01&chfieldto=Now&query_format=advanced&resolution=FIXED&resolution=INVALID&resolution=WONTFIX&resolution=DUPLICATE&resolution=WORKSFORME&resolution=MOVED